qcommon/src/main/java/jake2/qcommon/network/Netchan.java

```diff
@@ -110,7 +113,7 @@ public static void Netchan_Init() {
     }
 
     private static final byte send_buf[] = new byte[Defines.MAX_MSGLEN];
-    private static final sizebuf_t send = new sizebuf_t();
+    private static final sizebuf_t buffer = new sizebuf_t();
```

Comment:
Will `sizebuf_t` eventually be camelCased or renamed?

Reply:
there are many such cases in jake, i don't have a strategy for them yet, apart from "if class is completely rewritten, rename it"

Reply:
in fact it very closely resembles standard `OutputStream` or `ByteBuffer`